index.js

```diff
@@ -41,6 +41,13 @@ function HttpServerPort() {
                 }
             }
         },
+        publicRoutes: [
```

Comment:
should be publicMethods instead of publicRoutes as per the way the configuration is used in handlers.js